README.md

```diff
@@ -1,157 +1,163 @@
-# SignalFx-Tracing Library for Ruby: An OpenTracing Auto-Instrumentor
+# SignalFx Tracing Library for Ruby
 
-This utility provides users with the ability of automatically configuring OpenTracing community-contributed [instrumentation libraries](https://github.com/opentracing-contrib) for their Ruby application via a single function.
+The SignalFx Tracing Library for Ruby helps you instrument Ruby applications
+with the OpenTracing API to capture and report distributed traces to SignalFx.
 
-```ruby
-require 'signalfx/tracing'
+The library consists of an auto-instrumentor that works with OpenTracing
+community-provided instrumentations, and provides a bootstrap utility to help
+install instrumentations. It creates a Jaeger tracer to send trace data to SignalFx.
```

Comment:
`to help install your desired instrumentations.  It also configures and uses a [Jaeger tracer](https://github.com/salemove/jaeger-client-ruby) to send trace data to SignalFx.`